srcpkgs/nim/template

```diff
@@ -27,7 +27,7 @@ post_extract() {
 
 do_build() {
 	cd csources
-	CFLAGS= LDFLAGS= sh build.sh
+	CC=gcc LD=gcc CFLAGS= LDFLAGS= sh build.sh
```

Comment:
Necessary to get the linking to work correctly, and matching default build.sh behaviour.